psi4/src/psi4/occ/cc_energy.cc

```diff
@@ -110,90 +110,80 @@ void OCCWave::omp2_mp2_energy() {
 
     if (reference_ == "RESTRICTED") {
         // Same-spin contribution
-        global_dpd_->buf4_init(&Tss, PSIF_OCC_DPD, 0, ID("[O,O]"), ID("[V,V]"), ID("[O,O]"), ID("[V,V]"), 0,
-                               "TAA <OO|VV>");
+        std::string temp = (wfn_type_ == "OCEPA") ? "2" : (wfn_type_ == "OMP2") ? "" : "2_1";
+        std::string t_same = "T" + temp + "AA <OO|VV>";
+        global_dpd_->buf4_init(&T, PSIF_OCC_DPD, 0, ID("[O,O]"), ID("[V,V]"), ID("[O,O]"), ID("[V,V]"), 0,
+                               t_same.c_str());
         global_dpd_->buf4_init(&K, PSIF_LIBTRANS_DPD, 0, ID("[O,O]"), ID("[V,V]"), ID("[O,O]"), ID("[V,V]"), 0,
                                "MO Ints <OO|VV>");
-        Emp2AA = 0.5 * global_dpd_->buf4_dot(&Tss, &K);
-        global_dpd_->buf4_close(&Tss);
+        Emp2AA = 0.5 * global_dpd_->buf4_dot(&T, &K);
+        global_dpd_->buf4_close(&T);
 
-        Escsmp2AA = ss_scale * Emp2AA;
+        Escsmp2AA = 1.0 / 3.0 * Emp2AA;
         Escsnmp2AA = 1.76 * Emp2AA;
         Escsmimp2AA = 1.29 * Emp2AA;
```

Comment:
SCS(MI)-MP2 not in `energy()` list or printed out? I don't blame you -- it's basis zeta dependent (and even tq is different from t or q) -- I just want to check not an oversight.

Reply:
It was probably an oversight in the original `occ` code. `Escsmimp2` variables are set, but aren't used. Nothing in `occ` accounts for basis-set dependence of these quantities beyond the basis-set dependence of the spin components of the correlation energy.

If that's not correct, I'll just rip them out.

Reply:
it won't pain me if scsmi is ripped out.